src/routes/routes.accounts.ts

```diff
@@ -0,0 +1,59 @@
+import express from 'express';
+import {AccountsController} from '../http/controllers/controller.accounts';
+
+export const AccountsRoute = (app: express.Application) => {
+  /**
+   * * Get account by ID endpoint
+   * @param acccountID {Number}
+   * @protected Admin or User with accountID
+   */
+  app.get(`/:accountID`, AccountsController._getById);
```

Comment:
We can simplify access control by dividing this endpoint in two:

- /me/
   - Without parameters
   - Provides only data related to the authentication token account.
-  /:accountID/
   - Granted only for Admin
   - Provides data for parameter accout.

Then, wen can use this schema for accesing /me/dependents and so.

Reply:
Why? 
What do you think about it? Do you want to make a call to discuss it?

Reply:
Change approved :)